Jellyfin.Api/Controllers/VideosController.cs

```diff
@@ -54,6 +54,8 @@ public class VideosController : BaseJellyfinApiController
 
         private readonly TranscodingJobType _transcodingJobType = TranscodingJobType.Progressive;
 
+        private StreamState? _videoStreamState;
```

Comment:
What's the point of this?

Reply:
Progressive transcoding.
https://github.com/jellyfin/jellyfin/blob/2579b2db561566df9c171d714446d40f1b0bf23d/Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs#L103

Reply:
Can't you just remove the using keyword? But what is disposing it now then?

Reply:
You're right.😅 Removing the using is enough.